### PR TITLE
Json server connection

### PR DESCRIPTION
**Pull request by RobinBurri**
This feature just allows us to pass a simple json as an argument to the program and it will set the socket port to whatever  <"port" : value> is in the json file.

### CODE REVIEW

**Comment by t-h2o** [comment]
To testing:
1. change value in `config.json`
2. on the browser `localhost:port/index.html`, it's important to precise `/index.html` else that doesn't work

File: src/serverTest/serverTest.cpp [hunk 1]
```diff
@@ -15,8 +17,12 @@ serverTest()
 	 *	AF_INET + SOCK_STREAM always protocol = 0,
 	 *	address for this socket, let the OS choose = INADDR_ANY
 	 */
-	const int				 MAXLINE = 1000;
-	Socket					 sock(AF_INET, 8080, SOCK_STREAM, 0);
+
+	json::Value val(config);
+	const int	MAXLINE = 1000;
+	std::cout << "HERE " << std::endl;
+	double					 port = val.get("port").get<double>();
```
Comment (t-h2o) [inline src/serverTest/serverTest.cpp:24]: The port maximum is the same of the maximum port `~65000 ~= 2^16 - 1`
a short is 16 bit integer
```suggestion
	unsigned short					 port = val.get("port").get<double>();
```

Comment (RobinBurri) [reply src/serverTest/serverTest.cpp]: nice, thanks

**Review by t-h2o — APPROVED** [review]
Great 🚀

**Review by 0xCAF3D0OD — APPROVED** [review]
OK

**Pull request merged**